components.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "default",
+  "rsc": false,
+  "tsx": true,
+  "tailwind": {
```

Comment:
Should we still keep the `tailwind` field ?

Reply:
Oh! Removing it will make cli not working

<img width="1053" alt="image" src="https://github.com/fisand/unocss-preset-shadcn/assets/38493346/d141cd94-8121-40dc-b4e3-e420e49f4c37">